src/test/java/ca/islandora/syn/settings/SettingsParserDigestTest.java

```diff
@@ -5,49 +5,54 @@
 import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertTrue;
 
-import java.io.ByteArrayInputStream;
-import java.io.InputStream;
+import java.io.StringReader;
 
 import org.junit.Test;
 
 public class SettingsParserDigestTest {
 
     @Test
     public void testOneSitePath() throws Exception {
-        final String testXml = String.join("\n"
-            , "<config version=\"12\">"
-            , "  <site url=\"http://test.com\" algorithm=\"RS384\" path=\"test/path.key\" encoding=\"PEM\"/>"
-            , "</config>"
-        );
-
-        final InputStream stream = new ByteArrayInputStream(testXml.getBytes());
-        final Config settings = SettingsParser.getSitesObject(stream);
-        assertEquals(12, settings.getVersion());
+        final String testXml = String.join("\n",
+                "---",
+                "version: 1",
+                "site:",
+                "  url: http://test.com",
+                "  algorithm: RS384",
+                "  path: test/path.key",
+                "  encoding: PEM");
+
+        final StringReader stream = new StringReader(testXml);
+        final Config settings = SettingsParser.create(stream).getConfig();
+        assertEquals(1, settings.getVersion());
         assertEquals(1, settings.getSites().size());
 
         final Site site = settings.getSites().get(0);
         assertEquals("RS384", site.getAlgorithm());
         assertEquals("http://test.com", site.getUrl());
         assertEquals("test/path.key", site.getPath());
         assertEquals("PEM", site.getEncoding());
-        assertEquals("", site.getKey());
+        assertNull(site.getKey());
         assertFalse(site.getDefault());
     }
 
     @Test
     public void testOneSiteKey() throws Exception {
-        final String testXml = String.join("\n"
-                , "<config>"
-                , "  <site url=\"http://test.com\" algorithm=\"RS384\" encoding=\"PEM\" default=\"true\">"
-                , "multiline"
-                , "key"
-                , "  </site>"
-                , "</config>"
-        );
-
-        final InputStream stream = new ByteArrayInputStream(testXml.getBytes());
-        final Config settings = SettingsParser.getSitesObject(stream);
-        assertEquals(-1, settings.getVersion());
+        final String testXml = String.join("\n",
+                "---",
+                "version: 1",
+                "site:",
+                "  url: http://test.com",
+                "  algorithm: RS384",
+                "  encoding: PEM",
+                "  default: true",
+                "  key: |",
+                "    multiline",
+                "    key");
+
+        final StringReader stream = new StringReader(testXml);
```

Comment:
`StringReader` is `Autocloseable`; ideally this would be in `try-with-resource`.